pvlib/test/test_irradiance.py

```diff
@@ -61,6 +61,7 @@ def test_extraradiation(input, expected, method):
 
 
 @requires_numba
+@pytest.mark.xfail(raises=ValueError, reason = 'spa.calculate_deltat not implemented for numba yet')
```

Comment:
We can now get rid of the xfail on test_extraradiation_nrel_numba, right? It would be good to add a more localized test in test_spa.py or test_solarposition.py that does fail under the expected conditions (decorated with xfail, of course).